src/main/java/com/softserve/controller/AuthenticationController.java

```diff
@@ -118,4 +118,14 @@ public ResponseEntity getGoogleSignIn(HttpServletResponse response) throws IOExc
         response.sendRedirect(url + "oauth_login/google");
         return ResponseEntity.ok().body("Ok");
     }
+
+    @GetMapping("/google-error/{email}")
+    @ApiOperation(value = "Redirect with email", hidden = true)
+    public ResponseEntity<BadRequestEmailDTO> withResponseEntity(@PathVariable ("email") String email) {
```

Comment:
Use controller advice / handler instead of this

Reply:
done